example/src/test/scala/monocle/std/TreeExample.scala

```diff
@@ -42,4 +43,10 @@ class TreeExample extends MonocleSuite {
     universe(tree) shouldEqual Stream(tree, Leaf(2), Leaf(3))
   }
 
+  test("transformC transform Tree nodes counting number of changed nodes") {
```

Comment:
can you also write a property like: 
`for all xs: List[A], _1 compose transformC(Option)(xs) == (xs.size, xs)`